tests/lib/rules/no-useless-return.js

```diff
@@ -305,28 +350,10 @@ ruleTester.run("no-useless-return", rule, {
               }
             `
         },
-        {
-            code: `
-              function foo() {
-                try {
-                  foo();
-                  return;
-                } catch (err) {
-                  return 5;
-                }
-              }
-            `,
-            output: `
-              function foo() {
-                try {
-                  foo();
-                  
-                } catch (err) {
-                  return 5;
-                }
-              }
-            `
-        },
+
+        // FIXME: Re-add this case (removed due to https://github.com/eslint/eslint/issues/7481):
+        // https://github.com/eslint/eslint/blob/261d7287820253408ec87c344beccdba2fe829a4/tests/lib/rules/no-useless-return.js#L308-L329
+
```

Comment:
This fix caused this test case to fail due to https://github.com/eslint/eslint/issues/7481. However, the fix for https://github.com/eslint/eslint/issues/7481 will be semver-minor, so I think it's best to remove this case for now, and then re-add it when https://github.com/eslint/eslint/issues/7481 is fixed.